File: bernardo/Problema0327/problema0327.py
# Entrada de valores
lista = []
tamanho = 0
quantidade = 0

tamanho = int(input("Digite o número de funcionários da empresa: "))

for num in range(tamanho):
    funcionario = (
        str(input(f"Digite a profissão do {num + 1} funcionário: ")))
    lista.append(funcionario)

# Cálculo da quantidade de dentistas na empresa
for num in range(len(lista)):
    if lista[num].lower() == "dentista":
        quantidade += 1

# Saída
print(f"Quantidade de dentistas da empresa: {quantidade}")

File: bernardo/Problema0327/problema0327.java
import java.util.Scanner;

public class problema0327 {
    static Scanner input = new Scanner(System.in);

    public static void main(String[] args) {

        // Declaração de variávies

        // Entrada de valores
        String[] lista;
        int tamanho = 0;
        int quantidade = 0;

        System.out.println("Digite o número de funcionários da empresa: ");
        tamanho = Integer.valueOf(input.nextLine());

        lista = new String[tamanho];

        for (int i = 0; i < tamanho; i++) {
            System.out.println("Digite a profissão do " + (i + 1) + " funcionário: ");
            String funcionario = (input.nextLine());
            lista[i] = funcionario;
        }

        // Cálculo da quantidade de dentistas na empresa
        for (int i = 0; i < lista.length; i++) {
            if (lista[i].toLowerCase().equals("dentista")) {
                quantidade++;
            }
        }

        // Saída
        System.out.println("Quantidade de dentistas da empresa: " + quantidade);
    }
}
